### ISSUE TITLE
Paper Revision{2025.emnlp-main.882}

### ISSUE THREAD

**Issue by jy-evangeline** [acl-org/acl-anthology#9971]
### Anthology ID

[2025.emnlp-main.882](https://aclanthology.org/2025.emnlp-main.882/)

### Type of Change

Revision

### PDF of the Revision or Erratum

[_EMNLP_2025_05_19__Mixture_Reward_Learning.watermarked.pdf](https://github.com/user-attachments/files/32219058/_EMNLP_2025_05_19__Mixture_Reward_Learning.watermarked.pdf)

### Brief Description of Changes

This revision includes typo fix and notation simplification in the proof on page 17458 (Appendix A):
- corrects a typo in the definition of H on page 17458 (Appendix A), where the $\sigma_r$ should be $\sigma_k$ in the joint entropy. This is a typo rather than a mistake since from the second equation of the proof below it should be clear that we intended to use $\sigma_k$ rather than $\sigma_r$ in the definition of the joint entropy. 
- correct a typo in the detailed proof, instead of $s_k^* \sharp \pi$, which is the distribution of the Bradley-Terry logits, we intended to mean $(\sigma \circ s_k^*)\sharp\pi$, the distribution of the Bradley-Terry probability instead. This typo is also clear since later we explicitly wrote the tag “(TV distance of two Bernoulli)“. 
- simplify the pushforward notation in the theorem statement (page 17449 Theorem 3.2) and the proof since we find that the original notation is hard for many readers to understand. The semantic meaning remains the same.
Fixing these typos and notations help readers to fully understand and follow up our work. Thanks!
